src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -1212,43 +1212,17 @@ private static String getIntArrayPropertyValue(Field field, Object fieldValue) {
         }
 
         if (isPropertyTokenType(field)) {
-            boolean first = true;
-
+            final IntStream stream;
             if (value instanceof BitSet) {
-                final BitSet list = (BitSet) value;
-                final StringBuilder sb = new StringBuilder(20);
-
-                for (int i = 0; i < list.size(); i++) {
-                    if (list.get(i)) {
-                        if (first) {
-                            first = false;
-                        }
-                        else {
-                            sb.append(", ");
-                        }
-
-                        sb.append(TokenUtil.getTokenName(i));
-                    }
-                }
-
-                result = sb.toString();
+                stream = ((BitSet) value).stream();
             }
             else {
-                final StringBuilder sb = new StringBuilder(20);
-
-                for (int i = 0; i < Array.getLength(value); i++) {
-                    if (first) {
-                        first = false;
-                    }
-                    else {
-                        sb.append(", ");
-                    }
-
-                    sb.append(TokenUtil.getTokenName((int) Array.get(value, i)));
-                }
-
-                result = sb.toString();
+                stream = Arrays.stream((int[]) value);
             }
+            result = stream
+                    .mapToObj(TokenUtil::getTokenName)
+                    .sorted()
```

Comment:
I do not see sorting in original code.
Plus, 3 lines below in else block there is no sorting in stream.

Reply:
After the collection has changed from `List` to `Set`, the items will be iterated in a undetermined order.
We can keep items in a list in a specific order, but it is not possible for an unordered set. Using a `NavigableSet` is redundant here.
This will cause test failures because there are items in order in the docs. Sorting the items, we introduce an order here. In addition, it makes the docs more readable. The human brain is good at binary search.

Reply:
what about else block below where we have no sotring?

Reply:
There is no impact now, because this code get only empty arrays. But this is subject to change, so you are right, they should be changed too.

Reply:
done